.gitignore

```diff
@@ -1,5 +1,6 @@
 # local settings
 scripts/local_settings.py
+data/transcriptions.json
```

Comment:
we could probably ignore an entire `data/` directory; the csv files and other things I've downloaded for the NER experiment would go there to be ignored